ray-operator/config/crd/bases/ray.io_rayjobs.yaml

```diff
@@ -7075,6 +7075,12 @@ spec:
               submissionMode:
                 default: K8sJobMode
                 type: string
+              submitterConfig:
+                properties:
+                  submitterBackoffLimit:
+                    format: int32
+                    type: integer
+                type: object
```

Comment:
Should we default to 2 at the CRD level too?

Reply:
@jjyao you should add a kubebuilder marker and then generate CRD again.

https://github.com/ray-project/kuberay/blob/4836d0188d476c09e6fae34bb20739b8d6d43270/ray-operator/apis/ray/v1/rayjob_types.go#L78

Reply:
I'm able to set default for `BackOffLimit` but not the outer `SubmitterConfig` struct due to kubebuilder bug https://github.com/kubernetes-sigs/controller-tools/issues/622